sentry/src/main/java/io/sentry/DefaultSentryClientFactory.java

```diff
@@ -535,7 +535,7 @@ protected ContextManager getContextManager(Dsn dsn) {
             }
         }
 
-        return inAppPackages;
+        return Collections.unmodifiableList(inAppPackages);
```

Comment:
I think this kind of breaks the current API contract in a subtle way: while this is ABI compatible, subclasses will fail at runtime trying to add up to the base class collection of in-app packages.